src/components/AudioPlayer.tsx

```
@@ -0,0 +1,226 @@
+  id: string;
+  mediaUrl: string;
+  waveformUrl?: string;
+  durationSeconds: number;
+  aiMetadata?: {
+    summary?: string;
+  };
+}
+
+interface AudioPlayerProps {
+  post: AudioPost;
+  sound?: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)